### DISCUSSION TITLE
Render preview shows underline when underline emphasis is not enabled

### DISCUSSION THREAD

**Discussion by aubriewynbae** [requarks/wiki#3554] [Error / Bug Report]
**Describe the bug**
Render preview shows underscored words/phrases as underline rather than italicized, even when the "Underline Emphasis" option is turned off in "Rendering" options.

**To Reproduce**
1. Verify that "Underline Emphasis" option is turned off in "Rendering" section of Admin area.
2. Create a new page or edit and existing page
3. Tag word or phrase with underscores
4. Check render preview window to verify page will render correctly.

**Expected behavior**
Expecting the render preview to correctly show italicized when option is disabled.

**Screenshots**
![bugsnap-1](https://user-images.githubusercontent.com/69164296/93332840-74158300-f7f0-11ea-89f5-d3abf239d722.PNG)
![bugsnap-3](https://user-images.githubusercontent.com/69164296/93332844-7546b000-f7f0-11ea-8eee-b7f0e215fab8.PNG)

**Host Info (please complete the following information):**
 - OS: Ubuntu 20.04.1
 - Wiki.js version: 2.5.144
 - Database engine: pSQL 12.4

**Additional context:**
This does not affect actual rendering itself. I've tested a number of pages and they all rendered correctly (with italics rather than underline).

**Comment by NGPixel** [comment]
Rendering settings are not applied to preview, only the actual render when you save the page. This is coming in a future release.

**Comment by kneufeld** [comment]
~I'm seeing the quasi-opposite. Underscore'ing a word in the editing pane shows as italics but the preview and even the rendered version still shows as underlined.~

~I've tried with `underline emphasis` both on and off, same behaviour with both.~

I didn't use the "rendering reload" button. Disregard.

version `2.5.170`
